templates/uwsgi_systemd.service.erb

```diff
@@ -0,0 +1,15 @@
+#
+# This file is managed by puppet class 'uwsgi'
+#
+# uWSGI Emperor process systemd service
+#
+[Unit]
+Description=uWSGI Emperor
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/uwsgi --die-on-term --ini <%= @config_file %>
```

Comment:
I'll parameterize the binary path. On CentOS 7, the service failed to start without the full path.
Any preference for the name of the parameter? "app_binary_directory"?